05-Jason-Luo/wk02/5-fri/atm/js/atm.js

```diff
@@ -0,0 +1,199 @@
+//main objectives
+//deposit money
+//withdraw money
+//cant withdraw if not enough money for account balance
+//overdraft protection
+
+//deposit 
+//connect buttons to functions
+function checkForZero() {
+    $('.zero').removeClass('zero');
+    let checkingMoney = ($("#checking-balance").text());
+    console.log(checkingMoney);
+    let initMoney = parseInt(checkingMoney.substring(1, checkingMoney.length));
+    if (initMoney <= 0) {
+        $('#checking-balance').addClass('zero');
+    }
+
+    let savingsMoney = ($("#savings-balance").text());
+    console.log(savingsMoney);
+    let overallSavings = parseInt(savingsMoney.substring(1, savingsMoney.length));
+    if (overallSavings <= 0) {
+        $('#savings-balance').addClass('zero');
+    }
+
+}
+
+
+$(document).ready(function () {
+    checkForZero();
+    let checkingAccount = { balance: 2320 };
+    let savingAccount = { balance: 23255 };
```

Comment:
```suggestion
    let checkingBalance = 2320;
    let savingsbalance = 23255;
```
I don't see why these values need to be stored in objects.